db/models.py

```diff
@@ -50,3 +54,68 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(settings.AUTH_USER_MODEL,
+                             on_delete=models.CASCADE
+                             )
+
+    class Meta:
+        ordering = ["-created_at"]
+
+    def __str__(self) -> str:
+        return f"{self.created_at}"
+
+
+class Ticket(models.Model):
+    movie_session = models.ForeignKey(
+        MovieSession,
+        on_delete=models.CASCADE,
+        related_name="tickets"
+    )
+
+    order = models.ForeignKey(
+        Order,
+        on_delete=models.CASCADE,
+        related_name="order_tickets"
+    )
+    row = models.IntegerField()
+    seat = models.IntegerField()
+
+    def clean(self) -> None:
+        errors = {}
+
+        if not (1 <= self.row <= self.movie_session.cinema_hall.rows):
+            errors["row"] = [
+                f"row number must be in available range: "
+                f"(1, rows): (1, {self.movie_session.cinema_hall.rows})"
+            ]
+        if not (1 <= self.seat <= self.movie_session.cinema_hall.seats_in_row):
+            errors["seat"] = [
+                f"seat number must be in available range: "
+                f"(1, seats_in_row): "
+                f"(1, {self.movie_session.cinema_hall.seats_in_row})"
+            ]
+        if errors:
+            raise ValidationError(errors)
+
+    def save(self, *args, **kwargs) -> None:
+        self.full_clean()
+        super().save(*args, **kwargs)
+
+    class Meta:
+        constraints = [
+            UniqueConstraint(fields=["movie_session", "row", "seat"],
+                             name="unique_movie_session"),
+        ]
+
+    def __str__(self) -> str:
+        return (f"{self.movie_session.movie.title} "
+                f"{self.movie_session.show_time}"
+                f" (row: {self.row}, seat: {self.seat})")
+
+
+class User(AbstractUser):
+    pass
```

Comment:
Consider using `get_user_model()` method instead of directly referring to the `User` model. This is a good practice in Django as it allows for flexibility in case you decide to use a custom user model in the future.